### DISCUSSION TITLE
Next JS Node server deployment

### DISCUSSION THREAD

**Discussion by nebotaismael** [vercel/next.js#68704] [Help]
### Summary

Greetings, please i have completed developing my next js app and used npm run build and now i have the .next with these files 
![image](https://github.com/user-attachments/assets/8bceb133-773a-4689-9b4b-c312ccf3edf8)
the package.json only has 
![image](https://github.com/user-attachments/assets/cda4dfe0-5cfa-4675-b37f-93c7f7643f54)
, so how do I run this app on a node js server, also check out whats here 
![image](https://github.com/user-attachments/assets/be070aaf-6f95-40ba-ba25-ee79f744a9b0)
please this is a companies website im developing and i need to get it hosted , incase i want to run te _app.js, it says module not found

**Comment by icyJoseph** [comment]
You just need to run `npm run start`...

If you want to make it into a portable application, you can do the standalone mode. 

- https://nextjs.org/docs/pages/api-reference/next-config-js/output
